src/content/en/updates/2019/03/kv-storage.md

```diff
@@ -144,12 +153,13 @@ If you're familiar with using native JavaScript modules in browsers, you
 probably know that (at least up until now) importing anything other than a URL
 will generate an error. And `std:kv-storage` is not a valid URL.
 
-So that raises the question: _do we have to wait until all browsers support
-built-in module before we can use it in our code?_
+So that raises the question: _do we have to wait until all browsers support a
+built-in module before we can use it in our code?_ Thankfully, the answer is no!
 
-Thankfully, the answer is **no!** You can actually use built-in modules in your
-code today, with the help of another new feature called
-[import maps](https://github.com/WICG/import-maps).
+You can actually use built-in modules as soon as even one browser supports them
+thanks to the help of another feature we're
```

Comment:
I find the older wording more accurate; you don't need to wait for one browser.

Reply:
That's true. I can revert it.

Reply:
Originally I was thinking you'd never want to implement this before a single browser supports it. But that may not actually be true in the case of client work where you deliver a site that don't expect to change in the future. Using import maps preemptively may be a way to get future perf gains without having to update the site. (Assuming the name of the module doesn't change).